tests/foreman/api/test_activationkey.py

```diff
@@ -10,7 +9,24 @@
 from robottelo.test import APITestCase
 
 
-@ddt
+def _good_max_content_hosts():
+    """Return a generator yielding valid ``max_content_hosts`` values."""
+    return (gen_integer(*limits) for limits in ((1, 20), (10000, 20000)))
```

Comment:
Any reason for wrapping this statement in extra parenthesis? Just wondering

*commented on the commit instead of here